Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector.yml

```diff
@@ -0,0 +1,69 @@
+commonfields:
+  id: Github Event Collector
+  version: -1
+name: Github Event Collector
+display: Github Event Collector
+category: Analytics & SIEM
+description: Github logs event collector integration for XSIAM.
+configuration:
+- display: Server URL (e.g. 'https://api.github.com/orgs/XXXXX/audit-log')
+  name: url
+  defaultvalue: 'https://api.github.com/orgs/${ORGANIZATION}/audit-log'
+  type: 0
+  required: true
+- display: ""
+  displaypassword: API Key
+  name: auth_credendtials
+  type: 9
+  required: true
+  hiddenusername: true
+- display: The maximum number of logs to fetch.
+  name: limit
+  type: 0
+  defaultvalue: 1000
+  required: false
+- display: XSIAM update limit per request
+  name: events_to_add_per_request
+  type: 0
+  defaultvalue: 1000
+  required: false
```

Comment:
@adi88d @yaakovi 
Why have we needed this param? Did you encounter issues when sending large amounts of data to XSIAM?